### ISSUE TITLE
Qustion about your baseline

### ISSUE THREAD

**Issue by SakuBorder** [wadx2019/qvpo#2]
This is truly an impressive piece of work! I noticed that the available policy types in your code seem to include diffusion, mlp, and vae. I would like to ask if the MLP and VAE versions of your work will be open-sourced in the future? Thank you for your response, and I’m looking forward to your updates!

**Comment by wadx2019** [comment]
Thanks for your interest. This project is based on DIPO and the mlp and vae code is from this repo rather than written by us. So there is no mlp or vae version of our work. But I think it is easy for you to extend our method to them via mimicking the diffusion code.

**Comment by SakuBorder** [comment]
> Thanks for your interest. This project is based on DIPO and the mlp and vae code is from this repo rather than written by us. So there is no mlp or vae version of our work. But I think it is easy for you to extend our method to them via mimicking the diffusion code.

Thank you for the detailed explanation! We will try to extend your method to implement the MLP and VAE versions. Also, by the way, is using SAC directly similar to the MLP version?

**Comment by wadx2019** [comment]
No. SAC still utilizes the deterministic policy gradient to update the policy while QVPO updates the policy via action regression. This is the intrinsic difference between them.

**Comment by wadx2019** [comment]
So what you need to do is to use the mlp (Gaussian policy) to sample actions and choose good actions to do the regression.

**Comment by SakuBorder** [comment]
> So what you need to do is to use the mlp (Gaussian policy) to sample actions and choose good actions to do the regression.

Thank you for the detailed explanation! This clears up all my confusion.
